text/0000-image-retention-policy.md

```diff
@@ -0,0 +1,52 @@
+# Define an Image Retention Policy for Paketo Images
+
+## Summary
+
+At the moment, the Paketo Project is maintaining images (buildpack, builder, stack, etc..) since the beginning of the project. This RFC proposes that we define an image retention policy that would allow the project to delete old images.
+
+## Motivation
+
+Multiple reasons:
+
+1. It is impractical from a cost and resources perspective for the project to maintain all images throughout the history of time.
```

Comment:
I feel like we need some data to justify this assertion. Do we have a rough estimate of the costs to the project?